src/main/java/com/google/codeu/servlets/LocationsServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.google.codeu.servlets;
+import com.google.codeu.data.Datastore;
+import com.google.codeu.data.Location;
+
+import java.io.IOException;
+import com.google.gson.Gson;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+@WebServlet("/locations")
```

Comment:
I believe Allan originally made it "/location" not "/locations." I think it's best to keep it as "/location."

Reply:
Yes, and also modify the LocationServlet, not create a new class.